pkg/daemon/update.go

```diff
@@ -556,6 +613,136 @@ func (dn *Daemon) updateKernelArguments(oldConfig, newConfig *mcfgv1.MachineConf
 	return exec.Command("rpm-ostree", args...).Run()
 }
 
+// MountOSContainer mounts the container and returns the mountpoint
+func (dn *Daemon) MountOSContainer(container string) (mnt, containerName string, err error) {
```

Comment:
you can lowercase this method as it's not used outside `pkg/daemon` - `mountOSContainer`

Reply:
done